scripts/domain-expiry-check/domain_expiry_checker.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+# Specify the username
+username=""
+
```

Comment:
while I understand the logic, I think both this and the home_directory is useless, and should be removed. You can just add the correct location in domain_list variable.